browser/brave_profile_prefs.cc

```diff
@@ -182,7 +182,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   }
 
   // SKU SDK (can be used on account.brave.com)
-  brave_rewards::SkusSdkImpl::RegisterProfilePrefs(registry);
+  skus::SkusSdkContextImpl::RegisterProfilePrefs(registry);
```

Comment:
this should be a static method on the service as mentioned below

Reply:
Fixed; see earlier comment (moved into keyed service factory register method)
https://github.com/brave/brave-core/pull/10847/commits/780676bbebb2283387763229a4592c4eb4eadb9d